### PR TITLE
[Enhancement] Add FE tablet schedule to information_schema

### PR DESCRIPTION
**Pull request by decster**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Problem Summary(Required) ：

This PR adds FE tablet schedule information to information_schema, so as to debug tablet repair/balance problems more easily.  
Note that this information is only available in FE leader, so MySQL client should connect to FE leader to query this table. 

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [x] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Comment by decster** [comment]
run starrocks_fe_unittest

**Comment by decster** [comment]
run starrocks_fe_unittest

**Comment by decster** [comment]
run starrocks_fe_unittest

File: be/src/exec/schema_scanner/schema_fe_tablet_schedules_scanner.cpp [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        {"PRIORITY", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"STATE", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"STATUS", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"CREATE_TIME_MS", TYPE_BIGINT, sizeof(int64_t), false},
+        {"SCHEDULE_TIME_MS", TYPE_BIGINT, sizeof(int64_t), false},
+        {"FINISH_TIME_MS", TYPE_BIGINT, sizeof(int64_t), false},
+        {"CLONE_SRC_BE", TYPE_BIGINT, sizeof(int64_t), false},
+        {"CLONE_DEST_BE", TYPE_BIGINT, sizeof(int64_t), false},
+        {"CLONE_BYTES", TYPE_BIGINT, sizeof(int64_t), false},
+        {"CLONE_TIME_MS", TYPE_BIGINT, sizeof(int64_t), false},
```
Comment (chaoyli) [inline be/src/exec/schema_scanner/schema_fe_tablet_schedules_scanner.cpp:39]: `CLONE_TIME_MS` -> `CLONE_TIME` The granularity is second.

Comment (decster) [reply be/src/exec/schema_scanner/schema_fe_tablet_schedules_scanner.cpp]: rename to clone_time and change to double

**Review by chaoyli — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.4

**Pull request merged**